README.md

```diff
@@ -1 +1,78 @@
-# node-modules-folder-lab
\ No newline at end of file
+# Importing Folders (Proxy for Modules)
+
+## Objectives
+
+1. Implement a folder as a proxy for modules
+1. Create files in a folder, files with exports
+1. Import files/modules from another file via folder "proxy"
+
+## Introduction
+
+Code organization is important. If you want to really master Node and impress you co-workers, you need to know how to import folders. This powerful pattern is sometimes called a proxy for modules. This plugin pattern allows you to write a line of code to import a directory, which also recursively imports everything inside that directory. 
+
+Imagine you're working on a server logger. You must implement a series of modules which display your app logs in different colors. (It can be later used in a CLI game development too!).
+
+## Instructions
+
+1. Create a proxy file `index.js` in the `color-logs` directory.
+2. Import all the other JS files in the `color-logs` directory in the `index.js` (except `index.js` obviously)
+3. Export the imported file as an object. Follow the  `module.exports = {...}` pattern with filenames as values of properties (see extra info if this is too confusing)
+4. Create `main.js` in the root directory of this project and import the `color-logs` folder.
+5. Print two lines: "Access granted" in the green color and "Permission denied" in red.
+6. Bonus: refactor `index.js` to process an unlimited number of files from the `color-logs` folder with `fs` **without** using any extra npm modules like `require-dir`. You can use `readdirSync()` to read the list of files in the folder. Note: Don't forget to check that the files have `.js` extension and remember to convert any weird names (e.g., `deep-blue.js`) to proper JavaScript/Node names with camelCase (`deepBlue`).
```

Comment:
Strictly speaking, the weird names would be okay. Maybe offer an example about why it's a good idea to convert them to camelCase (e.g., calling `colors['deep-blue'](text)`, while possible, is a little funky).

I'm not sure that we need to include the phrase "proper JavaScript/Node names" -- we can just show students why they might prefer camelCase.